### PR TITLE
use !! prefix in networkd and timesyncd

### PR DESCRIPTION
**Pull request by yuwata**
Follow-up for #6577.

### CODE REVIEW

File: units/systemd-timesyncd.service.in [hunk 1]
```diff
@@ -11,7 +11,6 @@ Documentation=man:systemd-timesyncd.service(8)
 ConditionCapability=CAP_SYS_TIME
 ConditionVirtualization=!container
 DefaultDependencies=no
-RequiresMountsFor=/var/lib/systemd/clock
```
Comment (poettering) [inline units/systemd-timesyncd.service.in:14]: hmm, why drop this? If /var is split out we still need that... Hmm, or is this implied by StateDirectory=? IIRC it currently isn't but probably should be...

File: src/timesync/timesyncd-manager.c [hunk 2]
```diff
@@ -373,7 +373,7 @@ static int manager_adjust_clock(Manager *m, double offset, int leap_sec) {
                 return -errno;
 
         /* If touch fails, there isn't much we can do. Maybe it'll work next time. */
-        (void) touch("/var/lib/systemd/clock");
+        (void) touch("/var/lib/systemd/timesync/clock");
```
Comment (poettering) [inline src/timesync/timesyncd-manager.c:377]: there's one more mention of /var/lib/systemd/clock left: in the man page `man/systemd-timesyncd.service.xml`. That needs to be patched too.

**Comment by poettering** [comment]
(and sorry for the late review... I have been on vacation the past weeks)

File: units/systemd-timesyncd.service.in (hunk 1 above)
Comment (yuwata) [reply units/systemd-timesyncd.service.in:14]: Indeed. Updated.

File: src/timesync/timesyncd-manager.c (hunk 2 above)
Comment (yuwata) [reply src/timesync/timesyncd-manager.c:377]: Oh, thanks.

**Comment by yuwata** [comment]
Thank you for your comments. Updated.

**Comment by poettering** [comment]
Perfect!

**Comment by poettering** [comment]
CI failure appears unrelated.

**Pull request merged**